services/blockstorage/adapter/filesystem/file_system_persistence.go

```diff
@@ -84,6 +86,12 @@ func NewBlockPersistence(ctx context.Context, conf config.FilesystemBlockPersist
 		codec:        codec,
 	}
 
+	if fi, err := file.Stat(); err != nil {
+		adapter.logger.Info("unable to read file size for metrics", log.Error(err))
```

Comment:
@ronnno @jlevison do we want to return an error here? is this a valid case?

Reply:
don't think so - i cannot really point on when this line will ever run: if we cannot access the file, we have bigger problems (it will crash before) - the only scenario where this fails is if we cannot access the file.

Reply:
just for the sake of being uniform, I think it's better to return an error